src/components/AwardCard.vue

```diff
@@ -5,7 +5,7 @@
     :data-tournament="`${place}${suffix}`"
     :variant="isCard ? 'elevated' : 'text'"
   >
-    <div class="medal-icon d-flex align-center" :class="[isCard ? 'white' : 'cream']">
+    <div class="medal-icon d-flex align-center" :class="[isCard ? 'white px-5 py-5' : 'cream']">
```

Comment:
With the design change I think this container always has a surface-2 background. Similarly, I think the padding is probably fine to keep the same in the card + not-card contexts. Also 'pa' (short for padding-all) is a shorthand for px + py. So then this would simplify to 
```suggestion
    <div class="medal-icon d-flex align-center bg-surface-2 pa-5">
```